src/src_V2/modules/manager/container/sidebar/Sidebar.module.css

```diff
@@ -17,7 +17,7 @@
   left: 12px;
   width: 40px !important;
   height: 40px !important;
-  z-index: 1;
+  z-index: 11;
```

Comment:
@MMihran jan why we need this change here?

Reply:
@igortokhunts jan, we added z-index to sidebar, so our relative positioned container does not overflow it, and here the z-index of the button is changed so it is over sidebar